sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2013,6 +2014,21 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("SPARK-34157 Unify output of SHOW TABLES and pass output attributes properly") {
```

Comment:
We have unified tests for `SHOW TABLES` - `*.ShowTablesSuite`. Please, put related tests there.

Reply:
Thanks a lot!